src/browser/cdp/selectivity/utils.ts

```diff
@@ -0,0 +1,210 @@
+import { sortedIndex, memoize } from "lodash";
+import { SourceMapConsumer, type BasicSourceMapConsumer, type RawSourceMap } from "source-map";
+import fs from "fs";
+import path from "path";
+import { URL } from "url";
+import type { CDPRuntime } from "../domains/runtime";
+import type { CDPSessionId } from "../types";
+import { softFileURLToPath } from "../../../utils/fs";
+import type { NormalizedDependencies } from "./types";
+import { WEBPACK_PROTOCOL } from "./constants";
+
+export const fetchTextWithBrowserFallback = async (
```

Comment:
Pulling up sources with node.js "fetch" is faster, but doing it via browser is more reliable